src/components/JobsPanelParameters/JobsPanelParameters.js

```diff
@@ -290,8 +297,15 @@ const JobsPanelParameters = ({
     [panelState.tableData.parameters]
   )
 
+  const nameNotValid = name => {
+    return !!panelState.tableData.parameters.find(
```

Comment:
If all you want to know is whether an item exists in an array or not, use `Array.prototype.some` instead of: `Array.prototype.find` + convert to boolean

```suggestion
    return panelState.tableData.parameters.some(
```